bindings/java/wrapper.gradle

```diff
@@ -1,4 +1,4 @@
 task setup(type: Wrapper) {
     description = "Download the gradle wrapper and requisite files. Overwrites existing wrapper files."
-    gradleVersion = "5.0"
+    gradleVersion = "6.3"
```

Comment:
Do we need to require newer gradle? Does it mean projects using gradle 5.x can't use the binding?

Reply:
I don't think gradle is dependecy on its own. And Gradle 6 is needed for Java 13 support. Gradle 6.3 for Java 14 support.

Reply:
Shouldn't be a problem. Projects using an older version of Gradle can still use the binding.  They can just build this separately and then reference the jar in their build.